Lib/test/test_traceback.py

```diff
@@ -633,9 +636,14 @@ def f_with_multiline():
             'Traceback (most recent call last):\n'
             f'  File "{__file__}", line {self.callable_line}, in get_exception\n'
             '    callable()\n'
+            '    ~~~~~~~~^^\n'
             f'  File "{__file__}", line {lineno_f+1}, in f_with_multiline\n'
             '    if True: raise ValueError(\n'
-            '             ^^^^^^^^^^^^^^^^^'
+            '             ^^^^^^^^^^^^^^^^^\n'
+            '        "error over multiple lines"\n'
+            '        ^^^^^^^^^^^^^^^^^^^^^^^^^^^\n'
+            '    )\n'
```

Comment:
In cases like this, where there's only one character in one of the lines, it might appear off to a user, like it's a bug. Would it be more helpful to include line number in the beginning of the lines as well? Something like this maybe?

```python3
Traceback (most recent call last):
  File "/Users/lysnikolaou/repos/python/cpython/tmp/t.py", line 6, in <module>
  6.  multilineerror()
      ~~~~~~~~~~~~~~^^
  File "/Users/lysnikolaou/repos/python/cpython/tmp/t.py", line 2, in multilineerror
  2.  if True: raise ValueError(
               ^^^^^^^^^^^^^^^^^
  3.      "hello"
          ^^^^^^^
  4.  )
      ^
ValueError: hello
```

Reply:
I'm learning towards not doing this because I don't see much value - the user can infer which lines an error spans since the first lineno and the number of skipped lines, if applicable, are given. Adding this functionality would require a lot of changes to the test files.

Reply:
Yeah, let's get a simpler version for this PR as otherwise it will complicate even more the review and we are already dealing with a lot of code